### DISCUSSION TITLE
How Do I build this kind of trend chart using victory native

### DISCUSSION THREAD

**Discussion by crzycoder** [FormidableLabs/victory#2119] [Q&A]
# Bugs and Questions

### Checklist

- [x] This is not a `victory-native` specific issue. (Issues that only appear in `victory-native` should be opened [here](https://github.com/FormidableLabs/victory-native/issues/new))

- [x] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

- [x] I am using the latest version of Victory

- [x] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

Describe the issue you're seeing, and what you expect the behavior to be.

### Reproduction

Please try to reproduce the issue you're seeing in a sandbox. You can [fork this sandbox](https://codesandbox.io/s/m3xo745x2x) to get started.

If you can't reproduce your issue in a sandbox, please create a minimal git repo that demonstrates the problem you're seeing. Include instructions for installing and reproducing your error.

# Feature Requests

### Checklist

- [x] I've read through the [Docs](https://formidable.com/open-source/victory/docs) and [Guides](https://formidable.com/open-source/victory/guides) to make sure this functionality doesn't already exist

- [x] I've searched open issues to make sure I'm not opening a duplicate issue

### Description
![Trend](https://user-images.githubusercontent.com/43851308/137723125-ac6a4bde-01c2-4e20-b2df-af59ee019725.png)

**Comment by becca-bailey** [comment]
Hi @crzycoder, sorry we have been a little slow to respond to issues around here. I'm going to convert this issue to a discussion thread to prevent it from getting lost in issue reports. Thanks!
